app/channel/favorites/template.hbs

```diff
@@ -2,16 +2,31 @@
 
 	<section class="Section">
 		<div class="Container Container--full">
-			{{#list-manipulation list=model.favoriteChannels as |manipulatedList updateSorting|}}
-				<div class="Filters">
-					{{#btn-group class="Filter BtnGroup--right"}}
+			{{#list-manipulation
+					 list=model.favoriteChannels
+					 sortKey=sortKey
+					 as |manipulatedList updateSorting|}}
+				<div class="BtnGroups">
+					{{#btn-group}}
 						<span class="BtnGroup-label">Filter radio channels by</span>
```

Comment:
Aren't we sorting and not filtering? Thinking we should update the copy everywhere from filter->sort.